alert-policies/lampy/QuestionsPerSecond.yml

```diff
@@ -0,0 +1,40 @@
+# Name of the alert
+name: Question Per Second
+
+# Description and details
+description: |+
+  This alert is triggered when the current rate of Questions is greater than 4 for 5 minutes.
```

Comment:
before: `This alert is triggered when the current rate of Questions is greater than 4 for 5 minutes.`
can be changed to: `This alert is triggered when the current rate of questions exceeds 4 for 5 minutes.`